### ISSUE TITLE
Discord invite broken. Is this project still active? 

### ISSUE THREAD

**Issue by ingochris** [GooeyChickenman/victor#80]
Hi, is this project to reverse-engineer/save vector still active? The discord link is currently broken.

**Comment by MelanieT** [comment]
Yes but the discord got hacked a while back and the old links are no good anymore. I don't have a current one.

**Comment by rmcolbert** [comment]
New link: https://discord.gg/hcJZPvxBRP
